<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeOfferForm</class>
 <widget class="QDialog" name="TradeOfferForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>301</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxTradeRequest">
     <property name="title">
      <string>Trade Request</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>17</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFrom">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>From:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelFromPlayer">
        <property name="text">
         <string>&lt;Player&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTo">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>To:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelToPlayer">
        <property name="text">
         <string>&lt;Player&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelLumberOfferIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Lumber.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLumberOffer">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelClayOfferIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Clay.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelClayOffer">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWoolOfferIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Wool.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWoolOffer">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWheatOfferIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Wheat.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWheatOffer">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOreOfferIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Ore.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOreOffer">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>For (Wants):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelLumberWantIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Lumber.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLumberWant">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelClayWantIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Clay.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelClayWant">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWoolWantIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Wool.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWoolWant">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWheatWantIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Wheat.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelWheatWant">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOreWantIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/Resources.qrc">:/Images/Ore.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelOreWant">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offer (Gives):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonCounterOffer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Counter Offer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReject">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAccept">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/Resources.qrc"/>
  <include location="../../resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAccept</sender>
   <signal>clicked()</signal>
   <receiver>TradeOfferForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReject</sender>
   <signal>clicked()</signal>
   <receiver>TradeOfferForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
